screens/styles.js

```diff
@@ -0,0 +1,124 @@
+import { StyleSheet } from 'react-native';
+
+module.exports = StyleSheet.create({
+  containerHome: {
+    flex: 1,
+    alignItems: "center",
+    justifyContent: "center",
+  },
+  containerSignIn: {
```

Comment:
In a future PR, we should add more dividers to seperate which style is for what components. We should do it for sign in and sign up in a future PR

i.e) 
  // ==================================
  // Nutrition Card Styles
  // ==================================

but for sign in, sign up, and the components